ArcGISRuntimeSDKQt_CppSamples/Layers/ListKmlContents/ListKmlContents.cpp

```diff
@@ -189,24 +154,25 @@ void ListKmlContents::displayPreviousLevel()
 
   if (grandparentNode != nullptr)
   {
-    displayChildren(grandparentNode);
     m_currentNode = grandparentNode;
     emit currentNodeChanged();
     emit labelTextChanged();
```

Comment:
Does the `labelTextChanged` signal need to exist or can it be replaced by `currentNodeChanged` now?

Reply:
The same comment applies for `labelTextChanged`.

Reply:
both those signals are still needed because they are not always called whenever `currentNodeChanged` is emitted

Reply:
I'm unable to find a case where `emit labelTextChanged();` is not called immedialtey after `emit currentNodeChanged();`.

Reply:
Sometimes `emit currentNodeChanged()` is called but `emit labelTextChanged()` is not called